cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/AzureUtils.java

```diff
@@ -84,6 +85,10 @@ public class AzureUtils {
 
     private static final int MAX_DISK_ENCRYPTION_SET_NAME_LENGTH = 80;
 
+    private static final String AUTHORIZATION_FAILED_CODE = "AuthorizationFailed";
```

Comment:
are you sure, is this _always_ related to Marketplace image errors?

Reply:
Absolutley not, but combined with the Marketplace role in the message, it should be.